roles/sqs_queue/files/queue_attributes.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Policy": "{{sqs_queue.policy}}",
```

Comment:
defaults for all of these?

Reply:
Instead of defaults, I've removed the metadata and defined specific values in the attributes for each queue type. This is to prevent config drift across stacks.